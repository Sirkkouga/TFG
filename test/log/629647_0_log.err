INFO - 06/04/24 15:51:24 - 0:00:00 - Read 1071 words from the vocabulary file.

INFO - 06/04/24 15:51:24 - 0:00:00 - 9830 words (1071 unique) in 71 sentences.
INFO - 06/04/24 15:51:24 - 0:00:00 - 548 unknown words (40 unique), covering 5.57% of the data.
INFO - 06/04/24 15:51:24 - 0:00:00 - `: 52
INFO - 06/04/24 15:51:24 - 0:00:00 - insert@@: 48
INFO - 06/04/24 15:51:24 - 0:00:00 - TopLevelItem: 36
INFO - 06/04/24 15:51:24 - 0:00:00 - Items: 31
INFO - 06/04/24 15:51:24 - 0:00:00 - Clan: 28
INFO - 06/04/24 15:51:24 - 0:00:00 - job@@: 27
INFO - 06/04/24 15:51:24 - 0:00:00 - JSON@@: 25
INFO - 06/04/24 15:51:24 - 0:00:00 - ItemGroup: 25
INFO - 06/04/24 15:51:24 - 0:00:00 - fl@@: 25
INFO - 06/04/24 15:51:24 - 0:00:00 - clan: 25
INFO - 06/04/24 15:51:24 - 0:00:00 - Kil@@: 24
INFO - 06/04/24 15:51:24 - 0:00:00 - Job@@: 23
INFO - 06/04/24 15:51:24 - 0:00:00 - ClanPlay@@: 23
INFO - 06/04/24 15:51:24 - 0:00:00 - clude@@: 23
INFO - 06/04/24 15:51:24 - 0:00:00 - List@@: 21
INFO - 06/04/24 15:51:24 - 0:00:00 - ames: 17
INFO - 06/04/24 15:51:24 - 0:00:00 - riv@@: 14
INFO - 06/04/24 15:51:24 - 0:00:00 - q: 10
INFO - 06/04/24 15:51:24 - 0:00:00 - jenkins: 8
INFO - 06/04/24 15:51:24 - 0:00:00 - M: 7
INFO - 06/04/24 15:51:24 - 0:00:00 - ache: 6
INFO - 06/04/24 15:51:24 - 0:00:00 - enkins: 5
INFO - 06/04/24 15:51:24 - 0:00:00 - Play@@: 5
INFO - 06/04/24 15:51:24 - 0:00:00 - ws: 4
INFO - 06/04/24 15:51:24 - 0:00:00 - Vie@@: 4
INFO - 06/04/24 15:51:24 - 0:00:00 - ger: 4
INFO - 06/04/24 15:51:24 - 0:00:00 - ub@@: 4
INFO - 06/04/24 15:51:24 - 0:00:00 - K: 4
INFO - 06/04/24 15:51:24 - 0:00:00 - \: 2
INFO - 06/04/24 15:51:24 - 0:00:00 - Y: 2
INFO - 06/04/24 15:51:24 - 0:00:00 - Job completed successfully
INFO - 06/04/24 15:51:24 - 0:00:00 - Exiting after successful completion
