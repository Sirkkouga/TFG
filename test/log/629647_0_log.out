submitit INFO (2024-06-04 15:51:20,990) - Starting with JobEnvironment(job_id=629647, hostname=veuc09, local_rank=0(1), node=0(1), global_rank=0(1))
submitit INFO (2024-06-04 15:51:20,990) - Loading pickle: /home/usuaris/veu/javier.abella/TFG/test/log/629647_submitted.pkl
Saving the data to /home/usuaris/veu/javier.abella/TFG/test/java.test.cl.bpe.pth ...
submitit INFO (2024-06-04 15:51:24,081) - Job completed successfully
submitit INFO (2024-06-04 15:51:24,082) - Exiting after successful completion
